x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/string/Between.java

```diff
@@ -73,12 +73,12 @@ protected TypeResolution resolveType() {
             return resolution;
         }
 
-        resolution = isBoolean(greedy, sourceText(), Expressions.ParamOrdinal.FOURTH);
-        if (resolution.unresolved()) {
-            return resolution;
-        }
-
-        return isBoolean(caseSensitive, sourceText(), Expressions.ParamOrdinal.FIFTH);
+        return isBoolean(greedy, sourceText(), Expressions.ParamOrdinal.FOURTH);
+//        if (resolution.unresolved()) {
```

Comment:
leftover?

Reply:
No, I wanted to keep it there on purpose. The way the functions deal with case sensitivity is undecided still.